Dockerfile

```diff
@@ -6,14 +6,14 @@
 FROM ubuntu:16.04
 MAINTAINER Carl Masri <cmasri@uvic.ca>
 
-RUN apt-get update
-RUN apt-get install -y \
+RUN apt-get update && apt-get install -y \
```

Comment:
This was changed to comply with Dockerfile [best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get).